packages/app/package.json

```diff
@@ -182,6 +182,7 @@
     "posthtml-parser": "^0.4.1",
     "posthtml-render": "^1.1.0",
     "prism-react-renderer": "^1.0.2",
+    "protocol-handlers": "^0.1.2",
```

Comment:
Using the same library we use in V2 to detect protocol handlers that don't have any registered apps in the system.